getting_started/cavs-boot/cavs-dsp-boot-overview.rst

```diff
@@ -0,0 +1,126 @@
+
+Overview
+########
+
+There are two main DSP boot flows:
+
+* **Cold boot** performed when the host CPU exits an Sx state. FW binaries are
+  loaded into DSP memory and full state re-initialization is required. This
+  flow is also referred as *Purge Flow* in the figures below.
+
+* **RTD3 boot** when the DSP state is restored from the DSP internal memory.
+  This flow is available on platforms with access to Isolated Memory Region
+  (IMR) allocated for the DSP.
+
+IPC Communication with DSP ROM
+******************************
+
+Once the master DSP core (#0) is powered up and reset by the host driver, an
+IPC communication with the DSP ROM is required in order to set the boot
+options (see Boot Path Control Messages for details and list of platforms that
+require this step). It is a one-way message that does not require a response
+from the DSP.
+
+There may be some specific requirements about the order of the DSP core reset,
+sending IPC message, and the DSP core unstall operations. It is assumed that
+the following order is required unless specified otherwise by Boot Path
+Control Message in case of a specific platform:
+
+1.	Power up and reset the DSP Core 0,
```

Comment:
I've found it wise to avoid using tabs altogether.  Sometimes Sphinx/reST particularly when trying to get items aligned with consistent indenting.  ReST documentation recommends using spaces too (tab stops are assumed at every 8th column)